### PR TITLE
ibus: 1.5.13 -> 1.5.14

### PR DESCRIPTION
**Pull request by ericsagnes**
###### Motivation for this change

Update ibus to latest version.
release notes: https://github.com/ibus/ibus/releases/tag/1.5.14

This add a new emoji input mode, [details](https://fedoraproject.org/wiki/Changes/IBus_Emoji_Typing).

This also contains the following changes:
- added `nodePackages.emojione` (new dependency of ibus)
- added an `installCheck` to ibus
###### Things done
- [x] Tested using sandboxing
  ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS,
    or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file)
    on non-NixOS)
- Built on platform(s)
  - [x] NixOS
  - [ ] OS X
  - [ ] Linux
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

cc @ttuegel

### CODE REVIEW

File: pkgs/tools/inputmethods/ibus/default.nix [hunk 1]
```diff
@@ -29,12 +30,14 @@ stdenv.mkDerivation rec {
     "--disable-memconf"
     "--enable-ui"
     "--enable-python-library"
+    "--with-emoji-json-file=${nodePackages.emojione}/lib/node_modules/emojione/emoji.json"
```
Comment (Mic92) [inline pkgs/tools/inputmethods/ibus/default.nix]: Does this pull nodejs as a dependency?

Comment (ttuegel) [inline pkgs/tools/inputmethods/ibus/default.nix]: IIUC, it is a build-time dependency only. We should check it's not retained at run-time.

Comment (ericsagnes) [inline pkgs/tools/inputmethods/ibus/default.nix]: Just `emojione` and it actually use only one file from it.
Neither `emojione` or `nodejs` are in the `nix-store -qR` list.

**Comment by ericsagnes** [comment]
Thanks for merging!

**Pull request merged**